code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -40,7 +40,8 @@
 	update_icon()
 	PopulateContents()
 	if(mapload && !opened) //now takes it first because timery thingies does not work with these
-		take_contents()
+		spawn(0)
```

Comment:
sleep(0)

Reply:
What is the difference? I assume it performs the same purpose, but is it more optimized?

Reply:
sleep is apparently much better than spawn

Reply:
sleep(0) did nothing. spawn(0) is the way to go.

Reply:
alright then